etcdserver/membership/cluster.go

```diff
@@ -307,21 +307,21 @@ func (c *RaftCluster) RemoveMember(id types.ID) {
 	c.removed[id] = true
 }
 
-func (c *RaftCluster) UpdateAttributes(id types.ID, attr Attributes) bool {
+func (c *RaftCluster) UpdateAttributes(id types.ID, attr Attributes) {
```

Comment:
etcdserver doesn't need to call `s.cluster.SetVersion(semver.Must(semver.NewVersion(r.Val)))` on `UpdateAttributes` success?